app/controllers/cohorts/clients_controller.rb

```diff
@@ -396,6 +397,14 @@ def update
     end
 
     def pre_destroy
+      return unless @client.deleted?
+
+      @client.restore
```

Comment:
Why are we restoring here?

Reply:
This is what happens if you press delete on the deleted tab -- instead of the pre-destroy modal, it restores the client. Seemed simpler than adding a restore column, and route -- but, if you would prefer, I could do that -- it would also let us change the column header to be 'Restore'